PrivateCollections/Germany/Neukirchen006.xml

```diff
@@ -0,0 +1,502 @@
+                        name of God, Mary; single letters and words.</seg>
+                         <date notBefore="1850" notAfter="1930">.</date>
+                         <desc>Slender rounded script, irregular
+                           (size and slant change throughout the manuscript).
+                           The vowel markers of the 5th order are bulky,
+                           open. ም and ሞ look similar (both tend to a direct line),
+                           in many cases not easy to distinguish. </desc>
+                        <desc/>
+                     </handNote>
+
+                     <handNote xml:id="h2" script="Ethiopic" corresp="#ms_i6">
+                      <seg type="script">A script of a somewhat later
+                        period, possibly late 19th or early 20th century.</seg>
+                        <seg type="ink">Black, red</seg>
+                        <date notBefore="1850" notAfter="1930"/>
+                         <desc>Broad, rectangular; downstrokes are very broad.
+                           Irregular, crudely written.
+                           The vowel markers of the 5th order are bulky, open.</desc>
+                        <desc/>
+                     </handNote>
+
+                     <handNote xml:id="h3" script="Ethiopic" corresp="#a1">
+                      <seg type="script">A 20th-century script.</seg>
+                        <seg type="ink">Black, red.</seg>
+                        <date notBefore="1900" notAfter="1975"/>
+                         <desc>Crude, broadly spaced, strongly slanted; reddish brown ink used
+                         for rubrication.</desc>
+                         <note></note>
+                        <desc/>
+                        </handNote>
+
+                        <handNote xml:id="h4" script="Ethiopic" corresp="#a2">
+                         <seg type="script">A mid-19th- to early 20th-century script.</seg>
+                           <seg type="ink">Black, red.</seg>
+                           <date notBefore="1850" notAfter="1930"/>
+                            <desc>Very similar pr possibly identicaql to the main hand.</desc>
+                            <note></note>
+                           <desc/>
+                           </handNote>
+
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote type="ornamentation" xml:id="d1">
+                    <locus target="#3r"/>
+                    <desc>Bicoloure (red, black) ornamental band.</desc>
+                  </decoNote>
+                  </decoDesc>
+
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#159v"/>
+                        <desc type="GuestText"/>
+                        <ref type="work" corresp="LIT1860Malkea" >Malkǝʾa Mikāʾel</ref>
+                        <title type="incomplete"/>
```

Comment:
I think you can mark up everything with title, not ref @work. We also have this in the guidelines

Reply:
Ok, done.